package.json

```diff
@@ -60,6 +60,7 @@
     "memoizee": "^0.4.14"
   },
   "devDependencies": {
+    "@types/koa": "^2.0.49",
```

Comment:
Problem with this being a devDependency is that if anyone depends on `express-ajv-swagger-validation` types, their compilation is going to break unless they include `@types/koa` as their devDependency too. Including it as dependency would preserve it during runtime, so probably the least that could be done is readme updated to mention this dependency as necessary for Typescript builds.

Reply:
Thanks for pointing this out.

I'll just set it as `any`.

Reply:
Is it still needed?

Reply:
No it took me some time to edit from the phone 😂